src/mame/machine/gdrom.cpp

```diff
@@ -196,15 +209,20 @@ void gdrom_device::ExecCommand()
 
 				if (read_type != 2) // mode 1
 				{
-					fatalerror("GDROM: Unhandled read_type %d\n", read_type);
+					throw emu_fatalerror("GDROM: Unhandled read_type %d", read_type);
 				}
 
 				if (data_select != 2)   // just sector data
 				{
-					fatalerror("GDROM: Unhandled data_select %d\n", data_select);
+					throw emu_fatalerror("GDROM: Unhandled data_select %d", data_select);
 				}
 
-				printf("GDROM: CD_READ at LBA %x for %d blocks (%d bytes, read type %d, data select %d)\n", m_lba, m_blocks, m_blocks * m_sector_bytes, read_type, data_select);
+				LOGCMD("%s: CD_READ %02x %02x\n", tag(), command[2], command[4]);
+
+				LOGCMD("%s: CD_READ at LBA %x for %d blocks (%d bytes, read type %d, data select %d)\n",
+					tag(), m_lba, m_blocks,
+					m_blocks * m_sector_bytes, read_type, data_select
+				);
```

Comment:
More things redundantly adding the tag.  There are more of these further down, too.